<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ashish" failures="1" tests="1" name="Test(failed)(failed)" time="56.781" errors="0" timestamp="2021-12-02T10:39:19 IST">
  <testcase classname="testNg.testCase1" name="tc2" time="16.308">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Aptitude Questions and Answers - IndiaBIX] but found [Problems on Trains - Aptitude Questions and Answers]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Aptitude Questions and Answers - IndiaBIX] but found [Problems on Trains - Aptitude Questions and Answers]
at testNg.testCase1.tc2(testCase1.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
</testsuite> <!-- Test(failed)(failed) -->
